.vscode/settings.json

```diff
@@ -17,5 +17,6 @@
     "editor.formatOnSave": false
   },
   "tslint.autoFixOnSave": true,
-  "typescript.tsdk": "./node_modules/typescript/lib"
+  "typescript.tsdk": "./node_modules/typescript/lib",
+  "cSpell.enabled": false
```

Comment:
What is this prop?

Reply:
I was wondering about this and meant to ask before committing it. It seems like the artsy extension pack gave me a spell checker which is failing on `Formik.` I don't know why a spell checker is needed on ts files in the first place.

Reply:
I don't think it should be disabled, *it's really useful* - you should just add an exception for the word "formik" - it's with the lightbulb